Chatty Cathy/Chatty Cathy/HeaderView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  HeaderView.swift
+//  Chatty Cathy
+//
+//  Created by Artesia Ko on 2/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class HeaderView: UIView {
+    // MARK: View vars
+    var titleLabel: UILabel = {
```

Comment:
While this looks neat now, our convention for initializing views is to do it in the init() function or the viewDidLoad(). This method can get very large very quickly!